--- graphite2-1.3.1/src/CMakeLists.txt.orig	2015-09-05 14:35:43.749986744 +0200
+++ graphite2-1.3.1/src/CMakeLists.txt	2015-09-05 14:48:30.546621231 +0200
@@ -110,8 +110,7 @@
 if  (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
     set_target_properties(graphite2 PROPERTIES 
         COMPILE_FLAGS   "-Wall -Wextra -Wno-unknown-pragmas -Wendif-labels -Wshadow -Wctor-dtor-privacy -Wnon-virtual-dtor -fno-rtti -fno-exceptions -fvisibility=hidden -fvisibility-inlines-hidden -fno-stack-protector"
-        LINK_FLAGS      "-nodefaultlibs ${GRAPHITE_LINK_FLAGS}" 
-        LINKER_LANGUAGE C)
+        )
     if (CMAKE_COMPILER_IS_GNUCXX)
         add_definitions(-Wdouble-promotion)
     endif (CMAKE_COMPILER_IS_GNUCXX)
